### PR TITLE
Mrr Cookbooks

### PR DESCRIPTION
**Pull request by aabdelhak-EVS**
Notebooks on visualizing mrr data

### CODE REVIEW

File: notebooks/mrr/Crocus-Thredds-Cookbook.ipynb [hunk 1]
```diff
@@ -0,0 +1,2016 @@
+{
```
Comment (rcjackson) [inline notebooks/mrr/Crocus-Thredds-Cookbook.ipynb:1]: <p>Radar image not appearing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CROCUS-Urban/instrument-cookbooks/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (rcjackson) [inline notebooks/mrr/Crocus-Thredds-Cookbook.ipynb:1]: <p>Clear output of cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CROCUS-Urban/instrument-cookbooks/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

**Comment by aabdelhak-EVS** [comment]
I updated the cookbook

**Comment by aabdelhak-EVS** [comment]
Completed cookbook

**Pull request merged**